packages/webpack/src/runtime/runtime.js

```diff
@@ -261,6 +270,20 @@ const lavamoatRuntimeWrapper = (resourceId, runtimeKit) => {
     // wrap webpack runtime for policy check and hardening
     const policyRequire = wrapRequireWithPolicy(__webpack_require__, resourceId)
 
+    /**
+     * Wrapped chunk loader
+     *
+     * @param {string | number} chunkId
+     */
+    policyRequire.e = (chunkId) => {
+      // Chunk resolution at runtime may be a lookup table or a string concatenation. In the latter case it can be fooled into loading any file from under publicPath. That's why we only want to accept known chunks.
+      if (!LAVAMOAT.kch.includes(chunkId)) {
+        throw Error(
+          `Attempt to load a chunk that was not known at compile time: ${chunkId} from ${resourceId}`
```

Comment:
is this actionable? how?

Reply:
It gives you a chance to figure out what's happening if you have plugins that add chunks to your build later than we anticipated that to be possible. chunkIds are stable